### DISCUSSION TITLE
[Feature] Add a server switching function to the mobile client

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#11709] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I changed the server address, and the client has to wait for a long time to return to the server interface. I hope to add a function to switch server addresses in the settings.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2335
